docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -873,70 +885,45 @@ For all frameworks:
 You need to specify values for each input of the model. For more information, refer to the :doc:`Converting a Model to Intermediate Representation <openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model>` guide.
 
-.. _question-98:
+.. _question-102:
 
-Q98. What does the message "Operation _contrib_box_nms is not supported ..." mean?
+Q102. What does the message "Operation _contrib_box_nms is not supported ..." mean?
 #####################################################################################################################################################
 
 **A:** It means that you are trying to convert a topology contains the ``_contrib_box_nms`` operation which is not supported directly. However, the sub-graph of operations including ``_contrib_box_nms`` could be replaced with the DetectionOutput layer if your topology is one of the ``gluoncv`` topologies. Specify the ``--enable_ssd_gluoncv`` command-line parameter for Model Optimizer to enable this transformation.
 
-.. _question-99:
+.. _question-103:
 
-Q99. What does the message "ModelOptimizer is not able to parse *.caffemodel" mean?
+Q103. What does the message "ModelOptimizer is not able to parse *.caffemodel" mean?
 #####################################################################################################################################################
 
 **A:** If a ``*.caffemodel`` file exists and is correct, the error occurred possibly because of the use of Python protobuf implementation. In some cases, error messages may appear during model parsing, for example: "``utf-8`` codec can't decode byte 0xe0 in position 4: invalid continuation byte in field: mo_caffe.SpatialTransformerParameter.transform_type". You can either use Python 3.7 or build the ``cpp`` implementation of ``protobuf`` yourself for your version of Python. For the complete instructions about building ``protobuf`` from sources, see the appropriate section in the :doc:`Converting Models with Model Optimizer <openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide>` guide.
 
+.. _question-104:
 
-.. _question-100:
-
-Q100. What does the message "SyntaxError: 'yield' inside list comprehension" during MxNet model conversion mean?
-#####################################################################################################################################################
-
-**A:** The issue "SyntaxError: ``yield`` inside list comprehension" might occur during converting MXNet models (``mobilefacedet-v1-mxnet``, ``brain-tumor-segmentation-0001``) on Windows platform with Python 3.8 environment. This issue is caused by the API changes for ``yield expression`` in Python 3.8.
-The following workarounds are suggested to resolve this issue:
-1. Use Python 3.7 to convert MXNet models on Windows
-2. Update Apache MXNet by using ``pip install mxnet==1.7.0.post2``
-Note that it might have conflicts with previously installed PyPI dependencies.
-
-.. _question-101:
```

Comment:
why do you remove it? Looks still valid.

Reply:
We are no longer referring to this message in our code. I suspect this is because we always force user to use mxnet with 1.7.0.post2 or greated. But let me double check this